src/ResourceManagement/HDInsight/HDInsight/Generated/ClusterOperations.cs

```diff
@@ -43,6 +43,8 @@ namespace Microsoft.Azure.Management.HDInsight
     /// </summary>
     internal partial class ClusterOperations : IServiceOperations<HDInsightManagementClient>, IClusterOperations
     {
+        private const string _userAgentString = "ARM SDK v2.0";
```

Comment:
You need to add the changes here.  This PR will not be accepted without requested changes.